2023-10-18 13:39:36,537 INFO    StreamThr :10672 [internal.py:wandb_internal():86] W&B internal server running at pid: 10672, started at: 2023-10-18 13:39:36.536530
2023-10-18 13:39:36,538 DEBUG   HandlerThread:10672 [handler.py:handle_request():144] handle_request: status
2023-10-18 13:39:36,553 INFO    WriterThread:10672 [datastore.py:open_for_write():85] open: C:\Users\19233436\PycharmProjects\MOGE\wandb\run-20231018_133936-f5lmz9sh\run-f5lmz9sh.wandb
2023-10-18 13:39:36,553 DEBUG   SenderThread:10672 [sender.py:send():369] send: header
2023-10-18 13:39:36,585 DEBUG   SenderThread:10672 [sender.py:send():369] send: run
2023-10-18 13:39:37,083 INFO    SenderThread:10672 [dir_watcher.py:__init__():211] watching files in: C:\Users\19233436\PycharmProjects\MOGE\wandb\run-20231018_133936-f5lmz9sh\files
2023-10-18 13:39:37,083 INFO    SenderThread:10672 [sender.py:_start_run_threads():1100] run started: f5lmz9sh with start time 1697632776.538531
2023-10-18 13:39:37,090 DEBUG   SenderThread:10672 [sender.py:send_request():396] send_request: summary_record
2023-10-18 13:39:37,090 INFO    SenderThread:10672 [sender.py:_save_file():1354] saving file wandb-summary.json with policy end
2023-10-18 13:39:37,096 DEBUG   HandlerThread:10672 [handler.py:handle_request():144] handle_request: check_version
2023-10-18 13:39:37,097 DEBUG   SenderThread:10672 [sender.py:send_request():396] send_request: check_version
2023-10-18 13:39:37,142 DEBUG   HandlerThread:10672 [handler.py:handle_request():144] handle_request: run_start
2023-10-18 13:39:37,154 DEBUG   HandlerThread:10672 [system_info.py:__init__():31] System info init
2023-10-18 13:39:37,154 DEBUG   HandlerThread:10672 [system_info.py:__init__():46] System info init done
2023-10-18 13:39:37,154 INFO    HandlerThread:10672 [system_monitor.py:start():181] Starting system monitor
2023-10-18 13:39:37,154 INFO    SystemMonitor:10672 [system_monitor.py:_start():145] Starting system asset monitoring threads
2023-10-18 13:39:37,154 INFO    HandlerThread:10672 [system_monitor.py:probe():201] Collecting system info
2023-10-18 13:39:37,157 INFO    SystemMonitor:10672 [interfaces.py:start():190] Started cpu monitoring
2023-10-18 13:39:37,158 INFO    SystemMonitor:10672 [interfaces.py:start():190] Started disk monitoring
2023-10-18 13:39:37,158 INFO    SystemMonitor:10672 [interfaces.py:start():190] Started gpu monitoring
2023-10-18 13:39:37,162 INFO    SystemMonitor:10672 [interfaces.py:start():190] Started memory monitoring
2023-10-18 13:39:37,167 INFO    SystemMonitor:10672 [interfaces.py:start():190] Started network monitoring
2023-10-18 13:39:37,171 DEBUG   HandlerThread:10672 [system_info.py:probe():195] Probing system
2023-10-18 13:39:37,171 DEBUG   HandlerThread:10672 [system_info.py:_probe_git():180] Probing git
2023-10-18 13:39:37,174 ERROR   gpu       :10672 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-18 13:39:37,174 ERROR   gpu       :10672 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-18 13:39:37,225 DEBUG   HandlerThread:10672 [system_info.py:_probe_git():188] Probing git done
2023-10-18 13:39:37,225 DEBUG   HandlerThread:10672 [system_info.py:probe():240] Probing system done
2023-10-18 13:39:37,225 DEBUG   HandlerThread:10672 [system_monitor.py:probe():210] {'os': 'Windows-10-10.0.19045-SP0', 'python': '3.9.15', 'heartbeatAt': '2023-10-18T12:39:37.171691', 'startedAt': '2023-10-18T12:39:36.519533', 'docker': None, 'cuda': None, 'args': ('--algo', 'gpi_ls_discrete', '--env-id', 'deep-sea-treasure-v0', '--num-timesteps', '40000', '--gamma', '0.99', '--ref-point', '4000'), 'state': 'running', 'program': 'C:\\Users\\19233436\\PycharmProjects\\MOGE\\launch_experiment.py', 'codePath': 'launch_experiment.py', 'git': {'remote': 'https://github.com/JLu2022/MOGE.git', 'commit': 'caae01c0382da946b2eb1ca6e765981dbf8c2193'}, 'email': 'junlinlu16@gmail.com', 'root': 'C:/Users/19233436/PycharmProjects/MOGE', 'host': 'DESKTOP-UUPRB15', 'username': '19233436', 'executable': 'C:\\Users\\19233436\\Anaconda3\\envs\\MAPI\\python.exe', 'cpu_count': 16, 'cpu_count_logical': 24, 'cpu_freq': {'current': 2400.0, 'min': 0.0, 'max': 2400.0}, 'cpu_freq_per_core': [{'current': 2400.0, 'min': 0.0, 'max': 2400.0}], 'disk': {'total': 951.1328086853027, 'used': 196.70952606201172}, 'gpu': 'NVIDIA T1000', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA T1000', 'memory_total': 4294967296}], 'memory': {'total': 31.696025848388672}}
2023-10-18 13:39:37,225 INFO    HandlerThread:10672 [system_monitor.py:probe():211] Finished collecting system info
2023-10-18 13:39:37,225 INFO    HandlerThread:10672 [system_monitor.py:probe():214] Publishing system info
2023-10-18 13:39:37,225 DEBUG   HandlerThread:10672 [system_info.py:_save_pip():51] Saving list of pip packages installed into the current environment
2023-10-18 13:39:37,225 DEBUG   HandlerThread:10672 [system_info.py:_save_pip():67] Saving pip packages done
2023-10-18 13:39:37,226 DEBUG   HandlerThread:10672 [system_info.py:_save_conda():74] Saving list of conda packages installed into the current environment
2023-10-18 13:39:37,228 ERROR   HandlerThread:10672 [system_info.py:_save_conda():85] Error saving conda packages: [WinError 2] The system cannot find the file specified
Traceback (most recent call last):
  File "C:\Users\19233436\Anaconda3\envs\MAPI\lib\site-packages\wandb\sdk\internal\system\system_info.py", line 81, in _save_conda
    subprocess.call(
  File "C:\Users\19233436\Anaconda3\envs\MAPI\lib\subprocess.py", line 349, in call
    with Popen(*popenargs, **kwargs) as p:
  File "C:\Users\19233436\Anaconda3\envs\MAPI\lib\subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\19233436\Anaconda3\envs\MAPI\lib\subprocess.py", line 1420, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] The system cannot find the file specified
2023-10-18 13:39:37,229 DEBUG   HandlerThread:10672 [system_info.py:_save_conda():86] Saving conda packages done
2023-10-18 13:39:37,229 DEBUG   HandlerThread:10672 [system_info.py:_save_code():89] Saving code
2023-10-18 13:39:37,251 DEBUG   HandlerThread:10672 [system_info.py:_save_code():110] Saving code done
2023-10-18 13:39:37,251 DEBUG   HandlerThread:10672 [system_info.py:_save_patches():127] Saving git patches
2023-10-18 13:39:37,361 DEBUG   HandlerThread:10672 [system_info.py:_save_patches():169] Saving git patches done
2023-10-18 13:39:37,361 INFO    HandlerThread:10672 [system_monitor.py:probe():216] Finished publishing system info
2023-10-18 13:39:37,380 DEBUG   SenderThread:10672 [sender.py:send():369] send: files
2023-10-18 13:39:37,380 INFO    SenderThread:10672 [sender.py:_save_file():1354] saving file wandb-metadata.json with policy now
2023-10-18 13:39:37,381 INFO    SenderThread:10672 [sender.py:_save_file():1354] saving file code\launch_experiment.py with policy now
2023-10-18 13:39:37,381 INFO    SenderThread:10672 [sender.py:_save_file():1354] saving file diff.patch with policy now
2023-10-18 13:39:37,384 DEBUG   HandlerThread:10672 [handler.py:handle_request():144] handle_request: stop_status
2023-10-18 13:39:37,384 DEBUG   SenderThread:10672 [sender.py:send_request():396] send_request: stop_status
2023-10-18 13:39:37,620 DEBUG   SenderThread:10672 [sender.py:send():369] send: telemetry
2023-10-18 13:39:37,620 DEBUG   SenderThread:10672 [sender.py:send():369] send: metric
2023-10-18 13:39:37,620 WARNING SenderThread:10672 [sender.py:send_metric():1305] Seen metric with glob (shouldn't happen)
2023-10-18 13:39:37,620 DEBUG   SenderThread:10672 [sender.py:send():369] send: telemetry
2023-10-18 13:39:37,621 DEBUG   SenderThread:10672 [sender.py:send():369] send: telemetry
2023-10-18 13:39:37,621 DEBUG   SenderThread:10672 [sender.py:send():369] send: config
2023-10-18 13:39:37,621 DEBUG   SenderThread:10672 [sender.py:send():369] send: config
2023-10-18 13:39:37,685 INFO    wandb-upload_1:10672 [upload_job.py:push():113] Skipped uploading C:\Users\19233436\AppData\Local\Temp\tmpoq8sa6oywandb\sof1qcmh-code/launch_experiment.py
2023-10-18 13:39:37,949 INFO    wandb-upload_0:10672 [upload_job.py:push():133] Uploaded file C:\Users\19233436\AppData\Local\Temp\tmpoq8sa6oywandb\hpql6mh9-wandb-metadata.json
2023-10-18 13:39:38,036 INFO    wandb-upload_2:10672 [upload_job.py:push():133] Uploaded file C:\Users\19233436\AppData\Local\Temp\tmpoq8sa6oywandb\63i0uw6m-diff.patch
2023-10-18 13:39:38,089 INFO    Thread-16 :10672 [dir_watcher.py:_on_file_created():272] file/dir created: C:\Users\19233436\PycharmProjects\MOGE\wandb\run-20231018_133936-f5lmz9sh\files\output.log
2023-10-18 13:39:38,089 INFO    Thread-16 :10672 [dir_watcher.py:_on_file_created():272] file/dir created: C:\Users\19233436\PycharmProjects\MOGE\wandb\run-20231018_133936-f5lmz9sh\files\wandb-metadata.json
2023-10-18 13:39:38,089 INFO    Thread-16 :10672 [dir_watcher.py:_on_file_created():272] file/dir created: C:\Users\19233436\PycharmProjects\MOGE\wandb\run-20231018_133936-f5lmz9sh\files\diff.patch
2023-10-18 13:39:38,090 INFO    Thread-16 :10672 [dir_watcher.py:_on_file_created():272] file/dir created: C:\Users\19233436\PycharmProjects\MOGE\wandb\run-20231018_133936-f5lmz9sh\files\code\launch_experiment.py
2023-10-18 13:39:38,090 INFO    Thread-16 :10672 [dir_watcher.py:_on_file_created():272] file/dir created: C:\Users\19233436\PycharmProjects\MOGE\wandb\run-20231018_133936-f5lmz9sh\files\conda-environment.yaml
2023-10-18 13:39:38,090 INFO    Thread-16 :10672 [dir_watcher.py:_on_file_created():272] file/dir created: C:\Users\19233436\PycharmProjects\MOGE\wandb\run-20231018_133936-f5lmz9sh\files\wandb-summary.json
2023-10-18 13:39:38,090 INFO    Thread-16 :10672 [dir_watcher.py:_on_file_created():272] file/dir created: C:\Users\19233436\PycharmProjects\MOGE\wandb\run-20231018_133936-f5lmz9sh\files\requirements.txt
2023-10-18 13:39:38,090 INFO    Thread-16 :10672 [dir_watcher.py:_on_file_created():272] file/dir created: C:\Users\19233436\PycharmProjects\MOGE\wandb\run-20231018_133936-f5lmz9sh\files\code
2023-10-18 13:39:39,190 ERROR   gpu       :10672 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-18 13:39:39,191 ERROR   gpu       :10672 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-18 13:39:40,093 INFO    Thread-16 :10672 [dir_watcher.py:_on_file_modified():289] file/dir modified: C:\Users\19233436\PycharmProjects\MOGE\wandb\run-20231018_133936-f5lmz9sh\files\output.log
2023-10-18 13:39:41,204 ERROR   gpu       :10672 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-18 13:39:41,204 ERROR   gpu       :10672 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-10-18 13:39:42,007 DEBUG   HandlerThread:10672 [handler.py:handle_request():144] handle_request: status_report
2023-10-18 13:39:43,207 INFO    cpu       :10672 [interfaces.py:monitor():140] Process cpu has exited.
2023-10-18 13:39:43,241 DEBUG   SystemMonitor:10672 [system_monitor.py:_start():159] Starting system metrics aggregation loop
2023-10-18 13:39:43,241 DEBUG   SystemMonitor:10672 [system_monitor.py:_start():166] Finished system metrics aggregation loop
2023-10-18 13:39:43,241 DEBUG   SystemMonitor:10672 [system_monitor.py:_start():170] Publishing last batch of metrics
2023-10-18 13:39:43,246 ERROR   gpu       :10672 [interfaces.py:aggregate():161] Failed to serialize metric: division by zero
2023-10-18 13:39:43,246 DEBUG   SenderThread:10672 [sender.py:send():369] send: stats
2023-10-18 13:39:44,231 INFO    MainThread:10672 [internal.py:handle_exit():76] Internal process exited
